test/test_tensor_creation_ops.py

```diff
@@ -1155,7 +1155,6 @@ def test_cat2(self, device, dtype):
         self.assertEqual(z.size(), (21, SIZE, SIZE))
 
         self.assertRaises(RuntimeError, lambda: torch.cat([]))
-        self.assertRaisesRegex(TypeError, 'got None', lambda: torch.cat([x, None]))
```

Comment:
in general, you don't want to delete this -- you might want to change the type of error that is returned and update this test.

If the type of error changes or we go from error to no error, you should mark this change as bc breaking (label: topic: bc-breaking) and write a description oh what changed.

Reply:
The type of the test updated! thanks for guiding me :)